pkg/apis/kops/cluster.go

```diff
@@ -290,8 +290,9 @@ type ClusterSubnetSpec struct {
 	CIDR string `json:"cidr,omitempty"`
 
 	// ProviderID is the cloud provider id for the objects associated with the zone (the subnet on AWS)
-	ProviderID string `json:"id,omitempty"`
-
+	ProviderID string `json:"subnetId,omitempty"`
+	NgwId string `json:"ngwId,omitempty"`
```

Comment:
This is an API change - which means we need +1's from myself and @justinsb 

Cannot merge until we have a majority on this